platforms/core-configuration/model-core/src/testFixtures/groovy/org/gradle/api/internal/provider/TestCompoundAssignmentTransform.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal.provider;
+
+import org.codehaus.groovy.ast.ASTNode;
+import org.codehaus.groovy.control.CompilePhase;
+import org.codehaus.groovy.control.SourceUnit;
+import org.codehaus.groovy.transform.ASTTransformation;
+import org.codehaus.groovy.transform.GroovyASTTransformation;
+import org.gradle.groovy.scripts.internal.CompoundAssignmentTransformer;
+
+/**
+ * Applies compound operation transformer to a class or a method.
+ *
+ * @see org.gradle.api.internal.provider.support.SupportsCompoundAssignment
+ */
+@SuppressWarnings("unused")  // Applied through
```

Comment:
❔ My English is failing me. I don't understand the comment in this line.
Maybe, say that it's applied via `@TestCompoundAssignements` ?